bin/queuePopulator.js

```diff
@@ -60,6 +65,19 @@ async.waterfall([
         });
         done();
     },
+    done => {
+        healthServer.onReadyCheck(log => {
+            const state = queuePopulator.zkStatus();
+            if (state.code === zookeeper.State.SYNC_CONNECTED) {
+                return true;
+            }
+            log.error(`Zookeeper is not connected! ${state}`);
```

Comment:
minor: could be better to log state as a log object in second argument (with `{ state }`)